testcases/TC14.1-1-fail-1.md

```diff
@@ -0,0 +1,52 @@
+---
+# The layout must be 'testcase'; DO NOT Change
+layout: testcase
+
+# Brief, descriptive title for the test case
+title: Link Purpose(In Context) - Image/link has empty accessible name and description (TC14.1-1-fail-1)
+
+# The Test Case ID should follow the pattern: 
+# TC[Baseline Test Procedure #]-[Test Instruction #]-
+# [Expected Result (pass/fail/dna)]-[example #], e.g., TC05.1-1-fail-1
+tcid: TC14.1-1-fail-1
+
+# Description of the Test Case, the included code sample, test considerations,
+# and rationale for the expected result according to the applicable ICT
+# Baseline test
+descr: Image/link has empty accessible name and description.
+
+
+# Reference and link to the applicable ICT Baseline test
+app-basline:|[14.1 Test Procedure for Link Purpose (In Context)]https://section508coordinators.github.io/ICTTestingBaseline/14Links.html#141-test-procedure-for-link-purpose-in-context
+
+ 
+  **Baseline Test ID:** 14.1-LinkPurpose
+ 
+  **Test Instruction:** 1
+
+# Expected result that the ICT Baseline would predict
+# [Pass | Fail | DNA]
+result: Fail
+
+# Brief description of the rationale for the expected result
+result-descr: The code sample shows that the image/link has no accessible name or description.
+
+# URL for the code sample
+# In the sample code file, add id="tc_code" to the 
+# element that contains the relevant code snippet.
+#
+# Then upload the code sample to the 'testfiles' folder 
+# and provide the link (and only the url) below.
+sample: /testfiles/TF14/14.1-1-fail-1.html
+
+# Table of test instructions, including the following table headers: 
+# Test Instruction #; Instruction Detail; Expected Test Case Result
+#
+# Include the table in the content section below
+---
+| Test Instruction | Instruction Detail | Expected Test Case Result |
+|------------------|--------------------|---------------------------|
+| IC-1|All links including those that are scripted elements and assigned a role of a link.|
+| 14.1-1 | Check that the combination of accessible name and accessible description is not empty. [SC 4.1.2] | Fail | 
+| 14.1-2 | Check that the purpose of each link can be determined from any combination of the link text, accessible name and accessible description, and preceding page context (same paragraph, list, or table cell as the link or in a table header cell that is associated with the table cell that contains the link). [SC 2.4.4] | Fail |
```

Comment:
Only include the test instructions up to and including the one that produces the result for the test case. here, this test case is for instruction 1, so instruction 2 and on aren't needed. Delete instruction 2.

Reply:
Done.